### PR TITLE
Update vanilla v1.5.1

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Install vanilla-framework v1.5.1. Fix pattern updates in the pages. Remove unnecessary local Scss

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Click through the pages and check there is nothing broken
- Click small and medium screen

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2210
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2211
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2212

### CODE REVIEW

File: templates/cloud/training.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
   <div class="row">
     <div class="col-12">
       <hr />
-      <h3>On-site training</h3>
+      <h3 class="u-padding-bottom--large">On-site training</h3>
```
Comment (pmahnke) [inline templates/cloud/training.html]: I see why these are required.... but it's really inelegant.

File: templates/desktop/features.html [hunk 2]
```diff
@@ -49,7 +49,7 @@
   <div class="row">
     <div class="col-8">
       <h2>A whole world of apps</h2>
-      <p>Ubuntu offers thousands of apps available for download. Most are available for free and can be installed with just a few clicks.</p>
+      <p class="u-padding-bottom--x-large">Ubuntu offers thousands of apps available for download. Most are available for free and can be installed with just a few clicks.</p>
```
Comment (pmahnke) [inline templates/desktop/features.html]: again, this is inelegant

File: templates/download/alternative-downloads.html [hunk 3]
```diff
@@ -110,12 +110,14 @@ <h3 class="p-link--external"><span>Select the nearest mirror</span></h3>
 <div class="p-strip is-bordered">
   <div class="row">
     <div class="col-8">
-      <h2>Past ARM releases</h2>
-      <p>Find older versions of Ubuntu for specific ARM platforms.</p>
+      <div>
```
Comment (pmahnke) [inline templates/download/alternative-downloads.html]: I know why... but ugh.

**Comment by anthonydillon** [comment]
@pmahnke I know and will be removed as soon as the vertical spacing is fixed.

**Comment by spencerbygraves** [comment]
I've found 3 small things but otherwise looking good.

**Homepage**

- [ ] More vertical spacing needed under the title in the Insights section
- [ ] The charm image in the World's most popular... section is overlapping the section below

**Cloud page**

- [ ] A vertical gap is needed between the charm boxes in the Over 300 services... section

Thanks

**Comment by anthonydillon** [comment]
@spencerbygraves This is ready for another review.

**Comment by spencerbygraves** [comment]
Great, thanks @anthonydillon

**Pull request merged**